### PR TITLE
ED-3625: Angular migration from 14 to 15 

### PR DESCRIPTION
**Pull request by sedin-tushar**
- Angular Version Update: The Angular framework and dependencies has been updated from version 14 to version 15. 
- Removed enableIvy Configuration: In Angular version 15, Ivy is the default and only rendering engine. Therefore, the enableIvy configuration has been removed from the project configuration.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,21 +12,21 @@
...
-    "@angular-devkit/schematics": "^14.0.0",
-    "@angular/animations": "^14.0.0",
-    "@angular/common": "^14.0.0",
-    "@angular/compiler": "^14.0.0",
-    "@angular/core": "^14.0.0",
-    "@angular/forms": "^14.0.0",
-    "@angular/platform-browser": "^14.0.0",
-    "@angular/platform-browser-dynamic": "^14.0.0",
-    "@angular/router": "^14.0.0",
+    "@angular-devkit/architect": "^0.1500.0",
```
Comment (princegupta1131) [inline package.json]: Hi @sedin-tushar  can you please remove the ~, ^

File: projects/sb-dashlets/package.json [hunk 2]
```diff
@@ -1,29 +1,29 @@
 {
-  "name": "@project-sunbird/sb-dashlet-v14",
+  "name": "@project-sunbird/sb-dashlet-v15",
```
Comment (princegupta1131) [inline projects/sb-dashlets/package.json]: @sedin-tushar use this @project-sunbird/sb-dashlet don't use the  version
change the version to 8.0.0

File: projects/sb-dashlets/package.json [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
   "version": "1.0.6",
   "author": "Ravinder Kumar",
   "license": "MIT",
   "description": "A Simple extensible angular library to render JSON data into different chart formats",
   "dependencies": {
-    "@angular/common": "^14.0.0",
-    "@angular/core": "^14.0.0",
+    "@angular/common": "^15.2.10",
```
Comment (princegupta1131) [inline projects/sb-dashlets/package.json]: @sedin-tushar remove the ^

**Review by princegupta1131 — COMMENTED** [review]
update the version in readme file & raise pr in this branch https://github.com/Sunbird-Ed/sb-dashlets/tree/8.0.0_v15

File: projects/sb-dashlets/package.json (hunk 2 above)
Comment (sedin-tushar) [reply projects/sb-dashlets/package.json]: "name": "@project-sunbird/sb-dashlet",
  "version": "8.0.0","    @princegupta1131

**Pull request closed without merging**